ipv8/test/attestation/trustchain/test_community.py

```diff
@@ -572,6 +573,32 @@ def test_chain_crawl(self):
 
         self.assertEqual(self.nodes[1].overlay.persistence.get_number_of_known_blocks(), 4)
 
+    @inlineCallbacks
+    def test_chain_crawl_unknown_length(self):
+        """
+        Test crawling a chain with unknown length
+        """
+        def create_blocks(num):
+            self.nodes[0].endpoint.close()
+            key = default_eccrypto.generate_key(u'very-low').pub().key_to_bin()
```

Comment:
I'd like to phase out the use of the old M2Crypto format, could you use `u'curve25519'` here instead?

Reply:
Done